static/js/admin/src/action/SetAsPrimaryObjectResultAction.js

```diff
@@ -0,0 +1,66 @@
+define('SetAsPrimaryObjectResultAction', [ 'jquery', 'AjaxCallbackAction', "tpl!../templates/admin/editTypeForm"], function($, AjaxCallbackAction, editTypeTemplate) {
```

Comment:
can remove the editTypeForm/editTypeTemplate params

Reply:
done!